src/styles/_catalog.scss

```diff
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    transition: transform 0.3s, color 0.3s;
+
+    &:hover {
+      cursor: pointer;
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
I'm not sure if variables are necessary in such a small project, but I think they wouldn't hurt